core/domain/feedback_services.py

```diff
@@ -418,12 +418,14 @@ def _get_thread_from_model(thread_model):
     Returns:
         FeedbackThread. The corresponding FeedbackThread domain object.
     """
+    message_count = feedback_models.FeedbackMessageModel.get_message_count(
```

Comment:
We should only do this if the message_count is not present on the thread_model, right? Otherwise it involves an extra datastore GET?